### ISSUE TITLE
Cannot see through invisilble light if that wall is set to block sound

### ISSUE THREAD

**Issue by diwako** [manuelVo/foundryvtt-lichtgeschwindigkeit#16]
Recently started using both level and Lichtgeschwindigkeit. No issues when using the default levels setting which are 4 floors ranging from 0 to 39.

In one of my maps I added a river surrounded with cliffs and thought, if a player falls into that I could use Levels to indicate that. So I have created a new "floor" going from -10 to -1. Then however the shadow calculation broke with Lichtgeschindigkeit.

Versions:
```
foundry: 0.8.8 Standalone Windows App
Lichtgeschwindigkeit: 1.3.1
Levels: 1.1.2
Wall Height: 3.5.3.9
Better Roofs: 0.9.3.5
```

No errors in the console however.

Levels set up:
![image](https://user-images.githubusercontent.com/8259498/124379533-f9065600-dcb7-11eb-9688-bd8b2d3a8de6.png)

Wall set up, most of them are invisible walls due to fences and obstacles. that is Ground floor, ranging from 0 to 9
![image](https://user-images.githubusercontent.com/8259498/124379558-14716100-dcb8-11eb-8406-b5e889bc99e7.png)
Note: the invisible wall on the bride that would block players to walk onto it is at level -10 to -1, most of my walls are infinite up and down as they are are the same for all floors.

Foundry default renderer:
![image](https://user-images.githubusercontent.com/8259498/124379589-3d91f180-dcb8-11eb-828a-832bcbefb463.png)

Lichtgeschwindigkeit:
![image](https://user-images.githubusercontent.com/8259498/124379607-60bca100-dcb8-11eb-8942-ce29b73679d8.png)

Steps to reproduce:
- Load modules listed above.
- Create default floors with Levels
- create invisible walls at Ground floor level
- Create new floor going from -10 to -1
- Preview with character at ground floor level

**Comment by manuelVo** [comment]
Possibly related issue: https://github.com/theripper93/Levels/issues/23

**Comment by manuelVo** [comment]
Please do the following to help me debug this issue:
- Open the js console (it can be opened by pressing F12)
- Paste the following into the JS console: `CONFIG.debug.sightRays = true` and press enter
- Then move the token into a position where the bug occurs
- While the token is moving a yellow message containing garbled text will appear in the JS console. Please send me that text. It contains information that's vital for me in finding out what's causing this bug.

**Comment by diwako** [comment]
Sure thing. I moved one tile down and it spit out 3 console logs

https://pastebin.com/cAQi5ptC
![image](https://user-images.githubusercontent.com/8259498/124385403-b05d9580-dcd5-11eb-8690-fd3de01b79a8.png)

**Comment by diwako** [comment]
Looking at this debug draw, it might be related to walls that have infinite top and bottom values set.

![image](https://user-images.githubusercontent.com/8259498/124385470-177b4a00-dcd6-11eb-8d98-bfbf66a7f9e1.png)


Edit:
scratch that, first i looked at a wrong wall type and second once i checked an inviisble wall that is being rendered as a solid wall it was set to 0 to 9.

**Comment by manuelVo** [comment]
Thanks, I'll have a look

**Comment by manuelVo** [comment]
Hmm, according to the debug data all the walls in the scene are supposed to block vision. This means that something is going wrong before Lichtgeschwindigkeit even gets a chance to start the vision calculation. Unfortunately the debug data isn't sufficient in this case and I'll need the whole world data to see what's going on.

If you could send me the folder that contians a world with the scene that causes the bug (feel free to strip it of any assets and other scnees, if you wish to) as a zip file that would be great!

**Comment by diwako** [comment]
It is just this scene anyways, still at the beginning of my one shot preparation.

https://drive.google.com/file/d/1TwUncxHcx1m_YI2bCr-Y1fVPQNBpSTly/view?usp=sharing

used modules in this world:
![image](https://user-images.githubusercontent.com/8259498/124390371-8fa03a80-dceb-11eb-8841-e4e87ad12690.png)

**Comment by manuelVo** [comment]
Thanks to the world file you sent me I was able to reproduce this bug and found the issue and was able to find out what's happening. The cache that was introduced in 1.3.0 breaks Foundries 0.8 features (walls that block sound and roofs). I'm going to disable that cache for now to mitigate this issue.

**Comment by diwako** [comment]
Sad to hear the cache is getting disabled. Hopefully there is a way to fix this and have the performance with the cache again.

**Comment by manuelVo** [comment]
The cache being disabled isn't the end of the world performance-wise. Lichtgeschwindigkeit is still very fast. It's just falling back to v1.2 speeds for now. It could be that the cache is coming back at some point, but that'll require a significant amount of work, so it might not come back soon.

**Comment by manuelVo** [comment]
This issue is now fixed by a0df1cef867ac54aceeef0828cdac7bd8d41e9d2 and 13a533c9a569ad01c5adf2c4e95fdfadba00e66a.
